src/FixedDataTableBufferedRows.js

```diff
@@ -156,7 +163,10 @@ class FixedDataTableBufferedRows extends React.Component {
         />;
     }
 
-    return <div>{this._staticRowArray}</div>;
+    // We translate all the rows together with a parent div. This saves a lot of renders.
+    const style = {};
+    FixedDataTableTranslateDOMPosition(style, 0, props.offsetTop - (props.scrollTop % bufferHeight), false);
```

Comment:
From https://github.com/schrodinger/fixed-data-table-2/pull/432#discussion_r277478366
@somavara, @novakps , @mycrobe , any idea if this can cause floating point inaccuracies? I'm assuming (without much thinking) that as long as the errors always sum up to less than a unit, it won't be an issue..

Reply:
Not sure, I'll let others answer based on their thoughts.  One way to approach this which could be safer would be to first compute the parent shift.
`const parentShift = props.offsetTop - (props.scrollTop % bufferHeight)`
Then position the rows based off that as
`const rowOffsetTop = props.rowOffsets[rowIndex] - parentShift`

Would that be the same as the logic you have here?  It seems simpler in my mind.

Reply:
I'm seeing two problems with this parentShift approach. 
* Every time we scroll, the parent shift changes leading to a change in rowOffsetTop, which will cause a re-render. Our target is that rows' offset only depend on the row index so they won't rerender on scrolling.
* Once scrollTop passes bufferHeight, the scroll value is wrapped and starts over from 0 (due to the modulus operator). But since the row's offset isn't wrapped, they pass outside the table's limits.

Reply:
That makes sense.  Could we resolve those by adjusting the formula for parentShift?  My approach was to just use the formula you have here and base the rowOffsetTop off the parentShift to ensure it's accurate.

It's unclear to me why the same parentShift formula causes re-renders in one approach vs the other, but I don't think that's as important.  

I think what's important is that we base the rowOffsets on what they are supposed to be minus the parentShift offset.  Is there a way to do that and minimize re-renders?

Reply:
There's also a pretty good chance, I'm misunderstanding how this works and why.  If you feel that's the case could you write up a doc explaining how the parent & individual rows move during a scroll.  As well as why it's necessary for perf and how the math around it applies.

Reply:
Any update here Niranjan?  

Your [doc](https://docs.google.com/document/d/14YV3lwzxjpuiucdbtsCLElhDGL-LJtzMjlcwXXnrpJo/edit#) clarifies a lot.  Even if we lock parentShift to be in increments of 1000 (e.g. `const parentShift = Math.floor((props.offsetTop - (props.scrollTop % bufferHeight)) / 1000) * 1000;`)

Could we still have rowOffsetTop be `rowOffsetTop = props.rowOffsets[rowIndex] - parentShift`?

Reply:
I don't think so...
We need to do is make sure that 
* parent offset is wrapped
* row offset doesn't depend on parent offset, but rather on the increment index

Not sure how we can do it besides the index logic discussed in the doc. 
Anyone have any suggestions?